packages/orchestrator-ui-components/src/components/WfoSearchPage/WfoConditionRow/WfoConditionRow.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { useState } from 'react';
```

Comment:
I remember voting against but we have decided to use FC like this

```suggestion
import React, { FC, useState } from 'react';
```

See:
https://github.com/workfloworchestrator/orchestrator-ui-library/blob/51ea1a69c7cc1cf84557c6614b38f351251590f4/mkdocs/docs/adrs/0008-typescript-conventions.md